sunpy/coordinates/frames.py

```diff
@@ -587,6 +589,13 @@ def make_3d(self):
 
             d = np.fmin(d, dd) if self._spherical_screen['only_off_disk'] else dd
 
+        stack_trace = traceback.format_stack()
+        matching_string = 'wcsaxes.*_draw_grid'
```

Comment:
Since the longer string is `astropy/visualization/wcsaxes/coordinate_helpers.py", line 567, in _draw_grid`, to check for the method name and it being called in wcsaxes